packages/modules/viewer-react/src/components/app-ui/providers/ViewerStatusbarItemsProvider.tsx

```diff
@@ -8,44 +8,30 @@ import type {
   UiItemsProvider,
 } from "@itwin/appui-abstract";
 import { StatusBarSection } from "@itwin/appui-abstract";
-import { FooterSeparator } from "@itwin/appui-layout-react";
 import type { StatusBarItem } from "@itwin/appui-react";
 import {
-  FooterModeField,
   MessageCenterField,
   SelectionInfoField,
   SelectionScopeField,
   SnapModeField,
   StatusBarItemUtilities,
   TileLoadingIndicator,
   ToolAssistanceField,
-  withMessageCenterFieldProps,
-  withStatusFieldProps,
 } from "@itwin/appui-react";
 import * as React from "react";
 
 import type { ViewerDefaultStatusbarItems } from "../../../types";
 
 export class ViewerStatusbarItemsProvider implements UiItemsProvider {
   public readonly id = "ViewerDefaultStatusbar";
-  private _footerModeOnlySeparator: () => React.ReactNode;
 
-  constructor(private _defaultItems?: ViewerDefaultStatusbarItems) {
-    const FooterOnlyDisplay = withStatusFieldProps(FooterModeField);
-    this._footerModeOnlySeparator = (): React.ReactNode => {
-      return (
-        <FooterOnlyDisplay>
-          <FooterSeparator />
-        </FooterOnlyDisplay>
-      );
-    };
-  }
+  constructor(private _defaultItems?: ViewerDefaultStatusbarItems) {}
 
   public provideStatusBarItems(): CommonStatusBarItem[] {
     const items: StatusBarItem[] = [];
 
     if (!this._defaultItems || this._defaultItems.messageCenter) {
-      const MessageCenter = withMessageCenterFieldProps(MessageCenterField);
+      const MessageCenter = MessageCenterField;
```

Comment:
since all the HOCs we were using prior to the changes in this pr have been removed, i dont think we need to create an instance or alias it like this. no need for this var, just use MessageCenterField directly

Reply:
changed